spec/plans/import.fmf

```diff
@@ -27,6 +27,10 @@ description: |
     steps might be possible in the future but currently is not
     supported in any way.
 
+    Current only way to modify imported plan is via environment
+    variables. Variables defined in the plan override any variables
+    defined in the remote plan.
```

Comment:
Please, join the new paragraph with the preceding one and adjust `is not supported in any way` accordingly.

Reply:
Changed in amended commit.

Reply:
Thanks.